src/v3/src/transformer/selectAuthenticator/utils.ts

```diff
@@ -298,14 +298,16 @@ const getAuthenticatorButtonElements = (
 };
 
 export const getOVMethodTypeAuthenticatorButtonElements = (
-  options: IdxOption[],
+  authenticator: Input,
   step: string,
 ): AuthenticatorButtonElement[] => {
-  if (!options.length) {
+  const id = (authenticator.value as Input[])?.find(({ name }) => name === 'id')?.value as string;
```

Comment:
Is it possible for this value to be missing?

Reply:
From a real world standpoint, no, every authenticator should have an `id`, it only looks like this because of how the interface for Input is structured :(